Sources/PT.PM.Common/ConvertHelper.cs

```diff
@@ -0,0 +1,277 @@
+using System;
+using System.Globalization;
+using System.Numerics;
+using PT.PM.Common.Nodes;
+using PT.PM.Common.Nodes.Tokens;
+using PT.PM.Common.Nodes.Tokens.Literals;
+
+namespace PT.PM.Common
+{
+    public class ConvertHelper : ILoggable
+    {
+        private readonly RootUst root;
+
+        public ILogger Logger { get; set; } = DummyLogger.Instance;
+
+        public Language Language { get; }
+
+        public ConvertHelper(RootUst root)
+        {
+            this.root = root ?? throw new ArgumentNullException(nameof(root));
+            Language = root.Language;
+        }
+
+        public Token ConvertToken(ReadOnlySpan<char> span, TextSpan textSpan)
+        {
+            int spanLength = span.Length;
+            char firstChar = span[0];
+
+            if (spanLength == 1 && firstChar == '*')
```

Comment:
Looks like `&& firstChar == '*'` part is excessive here. Otherwise code in then branch makes little sense.